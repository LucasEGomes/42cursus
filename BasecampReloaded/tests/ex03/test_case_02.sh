#!/bin/sh
mkdir -p temp
mkdir -p temp/folder
touch temp/.sh
touch temp/find_.sh_me{0..2}.sh
touch temp/find_me{0..2}.sh.sh
touch temp/.find_me{0..2}.sh
touch temp/folder/.sh_find_me{0..2}.sh
touch temp/sh.sh.sh.sh
cp $ROOT_DIR/ex03/find_sh.sh temp/
cd temp
bash find_sh.sh | sort | cat -e > ../test_case_02.user
cd ..
diff test_case_02.expected test_case_02.user > test_case_02.output
if [ -z "$(cat test_case_02.output)" ];
then
    echo -n "O" >> $TEST_SUMMARY
else
    echo -n "X" >> $TEST_SUMMARY
fi
rm -rf temp/
