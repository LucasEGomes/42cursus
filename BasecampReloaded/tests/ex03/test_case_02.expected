$
.find_me0$
.find_me1$
.find_me2$
find_.sh_me0$
find_.sh_me1$
find_.sh_me2$
find_me0.sh$
find_me1.sh$
find_me2.sh$
find_sh$
folder/.sh_find_me0$
folder/.sh_find_me1$
folder/.sh_find_me2$
sh.sh.sh$
